tests/feature_selection/test_selection.py

```diff
@@ -3,28 +3,54 @@
 # Maximilian Christ (maximilianchrist.com), Blue Yonder Gmbh, 2016
 
 import numpy as np
-from unittest import TestCase
 import pandas as pd
```

Comment:
In this case, all tests in the file were part of the same class. Then I prefer not to have the class, as it doen't add any value. But you are right, they can help group similar tests. I will readd them now that there is another function tested in the same file.